### DISCUSSION TITLE
How use Morph relations

### DISCUSSION THREAD

**Discussion by MarcinStefaniak** [rappasoft/laravel-livewire-tables#892] [General]
Hi, 
How used morph relations in create Column? 

**I have below code in class ContractorTable**

`public function columns(): array
    {
        return [
            Column::make('Address', 'address.street'),
        ];
    }
    public function builder(): Builder
    {
        return Contractor::query()
            ->with('address')
            ->select();
    }`

**and in Model Contractor:**

`public function address(): MorphOne
    {
        return $this->morphOne(Address::class, 'addressable');
    }`

**I get error:**
Rappasoft\LaravelLivewireTables\Views\Column::setTable(): Argument #1 ($table) must be of type string, null given, called in /home/vagrant/code/th24/vendor/rappasoft/laravel-livewire-tables/src/Traits/Helpers/ColumnHelpers.php on line 24

**Comment by rappasoft** [comment]
There is currently limited relational support, I'm working on it.
